### DISCUSSION TITLE
namespace "std" has no member "cout" | Windows

### DISCUSSION THREAD

**Discussion by ItzKasmi** [microsoft/vscode-cpptools#10980] [Q&A]
Hi, I've been trying to troubleshoot this issue for the last hour. But can't seem to find out why I am getting this error. Provided below is my current code. Just trying to get C++ working on my local Windows environment.

![image](https://github.com/microsoft/vscode-cpptools/assets/95981007/464da2d6-79dc-4fca-9110-fa95f8e73332)

I can compile and run the code without errors but no matter what I have tried to change I am getting that squiggly error on the cout and anything following std:: for that matter.  I have read a lot of the previous solutions to this problem and nothing has worked for me.

Just hoping to get some help.

**Comment by sean-mcmanus** [comment]
You may need to set your compilerPath somehow, such as via setting the C_Cpp.default.compilerPath setting or via running the C/C++: Select IntelliSense Configuration command. Running the C/C++: Log Diagnostics command could also help diagnose which system headers are being picked up.
